test/core/GnosisSafe.Estimation.spec.ts

```diff
@@ -17,24 +17,23 @@ describe("GnosisSafe", async () => {
                 }
             }`
         const reverter = await deployContract(user1, reverterSource);
+        const decoderSource = `
+            contract Decoder {
```

Comment:
Why not a separate file, so it's easy to look for it?

Reply:
Because it is only used for testing here and it makes it easier to see what it does. Also the tests are then more complete in itself. I only put contracts for testing into separate files that require other contracts.